Safaa/setup.py

```diff
@@ -26,10 +26,10 @@
     packages=find_packages(where='src', ),
     package_dir={"": "src"},
     install_requires=[
-        'spacy>=3.0.0',
+        'spacy>=3.8.4',
         'joblib>=1.0.0',
-        'pandas>=1.1.0',
-        'scikit-learn>=1.3.0',
+        'pandas==2.2.3',
+        'scikit-learn==1.6.1',
```

Comment:
Should we use `==` for all dependencies instead of `>=` for `spacy` and `joblib`?

Reply:
sure, we can do that.